build.sbt

```diff
@@ -11,8 +11,6 @@ Global / resolvers += "Sonatype S01 OSS Snapshots" at "https://s01.oss.sonatype.
 
 ThisBuild / versionScheme := Some("early-semver")
 
-ThisBuild / scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
```

Comment:
I'd rather not lose this. It produces a warning which may now be a fatal warning. There are two fixes to that though:

1. We can convert the warnings to info level (there's a bit of sbt magic that does that but I'll have to look it up)
2. We can downgrade to version `0.5.0`.

Reply:
On testing, I've realised it's been removed because it is now built into Scalafix.